Cognite.Simulator.Tests/SeedData.cs

```diff
@@ -342,6 +342,85 @@ public static async Task<SimulatorRoutineRevision> GetOrCreateSimulatorRoutineRe
             Script = new List<SimulatorRoutineRevisionScriptStage>(),
         };
 
+        public static SimulatorModelRevisionDataFlowsheet SimulatorModelRevisionData = new SimulatorModelRevisionDataFlowsheet
+        {
```

Comment:
file becomes really big, maybe let's move flowsheet into `SeedDataFlowsheet.cs` or smth?

Reply:
or make a folder for it, with two files